src/allmydata/test/common_util.py

```diff
@@ -70,10 +69,11 @@ def flip_bit(good, which):
 def flip_one_bit(s, offset=0, size=None):
     """ flip one random bit of the string s, in a byte greater than or equal to offset and less
     than offset+size. """
+    precondition(isinstance(s, binary_type))
```

Comment:
No need to fix in this branch, but `isinstance(s, bytes)` works just as well and requires less post-python-2 cleanup, so maybe for future changes.